core/model/src/main/java/org/eclipse/rdf4j/model/ValueFactory.java

```diff
@@ -258,4 +258,14 @@ public default Statement createStatement(Resource subject, URI predicate, Value
 	public default Statement createStatement(Resource subject, URI predicate, Value object, Resource context) {
 		return createStatement(subject, (IRI) predicate, object, context);
 	}
+
+	/**
+	 * Creates a new triple with the supplied subject, predicate and object.
+	 *
+	 * @param subject   The statement's subject.
+	 * @param predicate The statement's predicate.
+	 * @param object    The statement's object.
+	 * @return The created triple.
+	 */
+	Triple createTriple(Resource subject, IRI predicate, Value object);
```

Comment:
> Let's make this a default method with the same impl as you currently have in AbstractValueFactory (I'd also leave that one in place, even though it doesn't do anything different) - that way this is backward compatible even for people who implement ValueFactory without using the abstract class.

That change would make the only SimpleXXX class that has a public constructor because it's in a different package. If you are ok with that I'll do it.

Reply:
Drat, forgot about that. Never mind then, leave as is.